crates/runtime/src/backtrace/aarch64.rs

```diff
@@ -0,0 +1,19 @@
+pub unsafe fn get_next_older_pc_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize).offset(1)
+}
+
+pub unsafe fn get_next_older_fp_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize)
+}
+
+pub fn reached_entry_sp(fp: usize, first_wasm_sp: usize) -> bool {
+    fp == first_wasm_sp - 16
+}
+
+pub fn assert_entry_sp_is_aligned(sp: usize) {
+    assert_eq!(sp % 16, 0, "stack should always be aligned to 16");
```

Comment:
I don't there is any need for a comment here - the assertion says everything that needs to be stated.